### ISSUE TITLE
[SDK][Defender] Importing hypercert-sdk into hypercerts-defender doesn't work

### ISSUE THREAD

**Issue by ryscheng** [hypercerts-org/hypercerts#362]
### What is it?

There's a graph build error?

**Comment by ryscheng** [comment]
Removing all dependencies on @hypercerts-org npm packages from defender at the moment to unblock. This is a nice to have, but not as important as the other issues

**Comment by ryscheng** [comment]
According to https://docs.openzeppelin.com/defender/autotasks#environment
we need to use rollup or webpack to bundle the SDK into our autotask.

**Comment by bitbeckers** [comment]
Spend a couple hours trying to get Defender to pack with the Hypercerts SDK. The blocker is that dependencies aren't resolved cleanly. This turned into a rabbithole between graph dependencies (solved in SDK 0.8.13) and `nft.storage`, `web3.storage` and `@web3-storage/multipart-parser`. 

```
Error: Cannot find module '@web3-storage/multipart-parser'
Require stack:
- /var/task/index.js
- /opt/nodejs/autotask-wrapper.js
- /var/runtime/index.mjs
```

Since this is a build issue in a restricted environment, this item is going back into the parking lot.  The improvement from efforts are that we use the ABI from `@hypercerts-org/contracts`
